docs/specs/sequences/endpoint_reconfiguration.md

```diff
@@ -22,11 +22,11 @@ sequenceDiagram
     participant D as Device
     participant E as Original Endpoint
     participant E' as New Endpoint
-    E->>D:CONFIG MESSAGE<br>blobset.blobs._iot_endpoint_config.base64 = <ENDPOINT><br>blobset.blobs._iot_endpoint.blob.phase = "final"
-    D->>E:STATE MESSAGE<BR>blobset.blobs._iot_endpoint_config.blob.phase = "apply"
+    E->>D:CONFIG MESSAGE<br>blobset.blobs._iot_endpoint_config.base64 = <ENDPOINT><br>blobset.blobs._iot_endpoint.phase = "final"
```

Comment:
pedantically there should be a check at the start that verifies that `state.blobset.blobs._iot_endpoint_config == null` (as the config block is empty so there should be no corresponding state).

Reply:
Added